project-progression-policy.md

```diff
@@ -44,8 +44,8 @@ and a vote by the Governing Board.
 #### Technical Review
 
 Projects are required to present their proposal at a TAC meeting.
-This proposal should cover all the items in the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md)
-and the [Technical Charter template](https://lists.confidentialcomputing.io/g/main/files/TAC/Project%20Submissions/Technical%20Charter%20%28custom+data%29%20--%20LF%20Projects,%20LLC%204-10-2019%20FINAL%20%2811%29.docx)).
+This proposal should cover all the items in the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md). The project should also complete the [LF Project Formation Intake Form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform), which will create the draft Technical Charter template, as well as other documents needed for approval. The project representative will receive a link to these drafts with a checklist of items to complete.
```

Comment:
Please line break markdown on sentences or at 100 character columns to facilitate diffs. (Though I recognize this file is inconsistent in that regard).
 ```suggestion
This proposal should cover all the items in the [Project Submission Template](https://github.com/confidential-computing/governance/blob/master/project-submission-template.md).
The project should also complete the [LF Project Formation Intake Form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform), which will create the draft Technical Charter template, as well as other documents needed for approval.
The project representative will receive a link to these drafts with a checklist of items to complete.
```